services/inference/tsfminference/dirutil.py

```diff
@@ -0,0 +1,41 @@
+# THIS FILE IS COPIED FROM THE BOILERPLATE DIRECTORY,
```

Comment:
should this be copied from boilerplate?

Reply:
I think it's probably going to have use in finetuning as well assuming we allow the finetuning of any model that a user might pre-train. If we are going to restrict finetuning to only base models we release then we can move it out of boilerplate.

Reply:
Yes, but does this particular file need to be tracked in git?

Reply:
Oh, sorry now I understand. No, that's a mistake. I'll remove it, thanks.